packages/fuels-test-helpers/src/lib.rs

```diff
@@ -134,6 +165,42 @@ pub async fn setup_test_client(
     (client, srv.bound_address)
 }
 
+#[cfg(not(feature = "fuel-core-lib"))]
```

Comment:
Shouldn't this be `#[cfg(feature = "fuel-core-bin")]`?

Reply:
I deleted ``fuel-core-bin`` flag and did as we agreed in the slack conversation.